modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/dataset_display/utilities/notifications.js

```diff
@@ -12,34 +12,18 @@
  * details.
  */
 
-export function showNotification(
-	message,
-	type = 'success',
-	closeable = true,
-	duration = 500
-) {
-	if (!window.AUI) {
-		return;
-	}
+import {openToast} from 'frontend-js-web';
 
-	AUI().use('liferay-notification', () => {
-		new Liferay.Notification({
-			closeable,
-			delay: {
-				hide: 5000,
-				show: 0,
-			},
-			duration,
-			message,
-			render: true,
-			title: Liferay.Language.get(type),
-			type,
-		});
+export function showNotification(message, type = 'success') {
+	openToast({
+		message,
+		title: Liferay.Language.get(type),
```

Comment:
I don't think this ever worked. Dynamic localization is not a thing we support.

We either need to get them from the server or to have a map with the posible type titles like:

```javascript
const TYPE_TITLE = {
    danger: Liferay.Language.get('error'),
    info: Liferay.Language.get('info'),
    success: Liferay.Language.get('success'),
}
```

Beware that titles are not always the same as type... I don't think we want to show "DANGER" in a toast :)

Reply:
Ouch.. Didn't notice that.

Yeah, will go with the map option. Thx